target/scripts/startup/setup-stack.sh

```diff
@@ -940,7 +940,6 @@ function _setup_security_stack
 #! /bin/bash
 
 sa-update --gpgkey 24C063D8 --channel kam.sa-channels.mcgrail.com && \
```

Comment:
`&& \` still relevant?

```suggestion
sa-update --gpgkey 24C063D8 --channel kam.sa-channels.mcgrail.com
```